docker-compose-fullstack.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
```

Comment:
1. I guess I thought this met a different need, I picture the `docker-compose.yml` file as the one you'd play with, since it's the one we encourage developing against (it exposes the ports and will have the data in it). If your goal is to just add in the website running, then it would be better to extend the `docker-compose.yml` that is already in the repository and suggest people run the full-stack via: `docker-compose -f docker-compose.yml -f docker-compose-fullstack.yml up`. Then this file becomes really simple with only the website definition and network.

2. Celery is running as part of the `devserver` command, you [see it here](https://github.com/pyslackers/website/blob/master/marketing/management/commands/devserver.py#L17). We don't have this deployed ATM, but it will be required for deploy. You can either add another service here, or change the command in the Dockerfile to be `./manage.py devserver` instead, or of course make a ticket to add it later.